tools/Google.Cloud.Tools.ReleaseManager/ContainerCommands/BuildRawCommand.cs

```diff
@@ -22,11 +22,9 @@ namespace Google.Cloud.Tools.ReleaseManager.ContainerCommands;
 /// <summary>
 /// Builds code generated for an API, without any other context (no handwritten code etc). Expected options:
 /// - generator-output: the directory containing the results of a previous generate command; required
-/// - api-path: (relative to api-root) e.g. google/cloud/functions/v2; required
+/// - api-path: (relative to generator-output) e.g. google/cloud/functions/v2; required
```

Comment:
Only for us - it's really relative to the API repository root, as specified in generate-raw. We happen to generate into a directory based on the API path (but could arguably generate directly into the output directory instead).

I'd suggest just changing this to "as provided to generate-raw".

I do wonder whether in the future we might end up combining the generate-raw and build-raw commands...

Reply:
Done. (Will push after you are done with the review in case there are other changes.)

And yes, maybe it makes sens to combine generate and build raw.